site/docs/v1/tech/apis/jwt.adoc

```diff
@@ -295,6 +295,9 @@ include::docs/v1/tech/apis/_standard-get-response-codes.adoc[]
 ==== Response Body
 
 [.api]
+[field]#refreshToken# [type]#[String]#::
+The refresh token. This will be the same as the [field]#refreshToken# provided.
```

Comment:
Depends on the Refresh Token Usage policy. If `Reusable` the same will be returned, if `OneTimeUse` then the updated token will be returned. Same goes for the cookie value below.

Reply:
Thanks, I updated.